composer.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "aligent/magento2-sitemap-extension",
+    "description": "Sitemap module for Magento 2",
+    "license": "GPL-3.0-only",
```

Comment:
Can you add a `LICENSE` file to the repository for this?

Reply:
@aligent-lturner Currently LICENSE file is already available in this repository.
Refer initial commit 
https://github.com/aligent/magento2-sitemap-extension/commit/e433f64fd4aae7e12b4e3906244e386da64e485a.